crates/deltalake-core/src/operations/write.rs

```diff
@@ -81,7 +89,7 @@ impl From<WriteError> for DeltaTableError {
 }
 
 /// Write data into a DeltaTable
-#[derive(Debug, Clone)]
+// #[derive(Debug, Clone)]
```

Comment:
I removed it because it was compiling when I changed a predicate type from String to Expression. I checked and and didn't find a place where those traits were used so instead of adding them to Expressions I thought it would make sense to remove them from here. I actually need to delete instead of commenting out
Besides, other builders don't have that either